cli/cli.ts

```diff
@@ -31,6 +31,15 @@ import * as crowdin from './crowdin';
 import * as youtube from './youtube';
 import { SUB_WEBAPPS } from './subwebapp';
 
+const Reset = "\x1b[0m";
+const Bright = "\x1b[1m";
+const FgRed = "\x1b[31m";
+const FgGreen = "\x1b[32m";
+const FgYellow = "\x1b[33m";
+const FgBlue = "\x1b[34m";
+const FgMagenta = "\x1b[35m";
+const FgCyan = "\x1b[36m";
```

Comment:
any reason not to just use chalk? that's what we use in the pxt-mkc

Reply:
Makes sense. Will use chalk.

Reply:
Downgraded chalk to v4 and still had to import using `require`. I don't see the utility of ESM modules.